packages/CodingStandard/README.md

```diff
@@ -38,6 +38,43 @@ $friends = [
 ```
 
 
+### Properties and constants must be separated by 1 blank line(s).
+
+- class: [`Symplify\CodingStandard\Fixer\ClassNotation\PropertyAndConstantSeparationFixer`](/src/Fixer/ClassNotation/PropertyAndConstantSeparationFixer.php)
+- This checker uses *[PHP-CS-Fixer](https://github.com/friendsofphp/php-cs-fixer)*
+
+:x:
+
+```php
+class SomeClass
+{
+    public $someProperty;
+    public $anotherProperty;
+}
+```
+
+:+1:
+
+```php
+class SomeClass
+{
+    public $someProperty;
+
+    public $anotherProperty;
+}
+```
+
+This checker requires 1 space by default. But if you need, you can **configure it**:
+
+
+```yaml
+# easy-coding-standard.neon
+checkers:
+    Symplify\CodingStandard\Fixer\ClassNotation\PropertyAndConstantSeparationFixer:
+        space_count: 2 # 1 by default
```

Comment:
@dg I've added checker that configures spaces between every constant and property.
It might be useful for Nette\CodingStandard.

Maybe I could add methods as well, since Nette has 2 spaces between methods.

What do you think?

Reply:
I feel that the spaces between the methods are already (somehow) checked.